src/components/metadata/listing/index.js

```diff
@@ -0,0 +1,352 @@
+
+    const { t } = useTranslation("metadata");
+    const classes = useStyles();
+
+    return (
+        <Toolbar>
+            {editable && (
+                <IconButton
+                    id={build(parentID, ids.BUTTONS.ADD)}
+                    color="primary"
+                    aria-label={t("addMetadata")}
+                    onClick={onAddAVU}
+                >
+                    <ContentAdd />
+                </IconButton>
+            )}
+            <div className={classes.title}>
+                <Typography id={build(parentID, ids.TITLE)} variant="h6">
+                    {t("avus")}
+                </Typography>
+            </div>
+        </Toolbar>
+    );
+};
+
+MetadataGridToolbar.propTypes = {
+    onAddAVU: PropTypes.func.isRequired,
+};
+
+const columnData = (t) => [
+    {
+        id: build(ids.COL_HEADER, ids.AVU_ATTR),
+        key: "attr",
+        name: t("attribute"),
+        enableSorting: true,
+    },
+    {
+        id: build(ids.COL_HEADER, ids.AVU_VALUE),
+        key: "value",
+        name: t("value"),
+        enableSorting: true,
+    },
+    {
+        id: build(ids.COL_HEADER, ids.AVU_UNIT),
+        key: "unit",
+        name: t("metadataUnitLabel"),
+        enableSorting: true,
+    },
+    {
+        id: build(ids.COL_HEADER, ids.AVU_AVUS),
+        key: "avus",
+        name: t("metadataChildrenLabel"),
+        enableSorting: true,
+        numeric: true,
+        padding: "none",
+    },
+    {
+        id: build(ids.COL_HEADER, ids.COL_ACTIONS),
+        key: "actions",
+        enableSorting: false,
+        padding: "none",
+    },
+];
+
+const AVURow = ({
+    editable,
+    selectable,
+    rowID,
+    selected,
+    attr,
+    value,
+    unit,
+    avuChildCount,
+    onRowSelect,
+    onRowEdit,
+    onRowDelete,
+}) => {
+    const { t } = useTranslation("metadata");
+    const classes = useStyles();
+
+    return (
+        <TableRow hover tabIndex={-1} selected={selected}>
+            {selectable && (
+                <TableCell padding="checkbox">
+                    <DECheckbox checked={selected} onChange={onRowSelect} />
+                </TableCell>
+            )}
+            <TableCell component="th" scope="row">
+                {attr}
+            </TableCell>
+            <TableCell>{value}</TableCell>
+            <TableCell>{unit}</TableCell>
+            <TableCell padding="none" align="right">
+                {avuChildCount}
+            </TableCell>
+            <TableCell padding="none">
+                <Grid
+                    container
+                    spacing={0}
+                    wrap="nowrap"
+                    direction="row"
+                    justify="center"
+                    alignItems="center"
+                >
+                    <Grid item>
+                        <IconButton
+                            id={build(rowID, ids.BUTTONS.EDIT)}
+                            aria-label={t("edit")}
```

Comment:
Should this `aria-label` text change depending on whether this item is `editable` or not?